tests/evaluators/test_evaluator_schemas.py

```diff
@@ -343,7 +345,9 @@ def test_config_validation_exact_match(self) -> None:
             "case_sensitive": True,
             "default_evaluation_criteria": {"expected_output": "test"},
         }
-        evaluator = ExactMatchEvaluator.model_validate({"config": config_dict})
+        evaluator = ExactMatchEvaluator.model_validate(
+            {"config": config_dict, "id": str(uuid.uuid4())}
```

Comment:
Here you are putting a guid ? I see the change to use guid at multiple places too